Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -132,6 +149,17 @@ - (id)initWithAnalytics:(SEGAnalytics *)analytics httpClient:(SEGHTTPClient *)ht
 static CTTelephonyNetworkInfo *_telephonyNetworkInfo;
 #endif
 
+#if TARGET_OS_OSX
+void get_platform_uuid(char * buf, int bufSize)
+{
+    io_registry_entry_t ioRegistryRoot = IORegistryEntryFromPath(kIOMasterPortDefault, "IOService:/");
+    CFStringRef uuidCf = (CFStringRef) IORegistryEntryCreateCFProperty(ioRegistryRoot, CFSTR(kIOPlatformUUIDKey), kCFAllocatorDefault, 0);
+    IOObjectRelease(ioRegistryRoot);
+    CFStringGetCString(uuidCf, buf, bufSize, kCFStringEncodingMacRoman);
+    CFRelease(uuidCf);
+}
+#endif
```

Comment:
Question: is this implementation going to generate a unique value to identify the computer?

Reply:
Yes, correctly. get mac uuid like ios uuid